db/seeds.rb

```diff
@@ -5,3 +5,31 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+Pet.destroy_all
+Shelter.destroy_all
+
```

Comment:
Should we add here on line 11? 
 
- `ApplicationPet.destroy_all`
- `Application.destroy_all`
- `Review.destroy_all`

Reply:
Ooh i think we probably should! Want to add these when we connect via zoom after office hours?

Reply:
@ttakahashi1591  updated!

Reply:
Perfect thanks!